### PR TITLE
change the module so that the zkwasm-service-helper in the frontend w…

### PR DESCRIPTION
**Pull request by Deepakraja03**
…ithout error

### CODE REVIEW

**Comment by Deepakraja03** [comment]
The zkwasm-service-helper error in the front-end is resolved. Now it can be called directly without calling it relatively

**Comment by Thirumurugan7** [comment]
Can you attach Error it showed first

**Comment by Deepakraja03** [comment]
I have attached the Error and changes made with this message
[Zkspin SDK Update Evidence.pdf](https://github.com/user-attachments/files/16647823/Zkspin.SDK.Update.Evidence.pdf)

File: sdk/gameplay/export/spinjs/src/index.ts [hunk 1]
```diff
@@ -2,3 +2,4 @@
 export * from "./GamePlay.js";
 export * from "./Proof.js";
 export * from "./Spin.js";
+export type { TaskStatus } from "zkwasm-service-helper";
```
Comment (tohacking) [inline sdk/gameplay/export/spinjs/src/index.ts:5]: Nit, add blank line here

File: sdk/gameplay/export/spinjs/tsconfig.json [hunk 2]
```diff
@@ -15,5 +15,8 @@
     "declaration": true,
   },
   "include": ["src/**/*", "test/"],
-  "exclude": ["node_modules", "dist"]
+  "exclude": ["node_modules", "dist"],
+  "paths": {
+      "zkwasm-service-helper": ["node_modules/zkwasm-service-helper/src/index.ts"]
+    }
 }
```
Comment (tohacking) [inline sdk/gameplay/export/spinjs/tsconfig.json:22]: Nit, add blank line here

**Review by tohacking — APPROVED** [review]
@Deepakraja03 LGTM. 

Thanks for the fix. I'll update the npm package after this is merged. 

Side words:
We'll want to eventually remove the dependency to zkwasm-service-helper. To do that, we'll need first to add an interface for generating proof for different zk providers.

**Pull request closed without merging**